firebase-perf/src/test/java/com/google/firebase/perf/metrics/validator/FirebasePerfTraceValidatorTest.java

```diff
@@ -169,7 +169,7 @@ public void screenTrace_shouldNotAllowNonPositiveTotalFrames() {
   }
 
   @Test
-  public void testInvalidCustomAttribute() {
+  public void traceValidator_invalidCustomAttribute_marksPerfMetricInvalid() {
```

Comment:
The best practice is to test one behavior per test. We should try to avoid having one monolithic test that asserts so many things.

Reply:
Good point. Separated the tests to individual atomic tests.